adafruit_ht16k33/segments.py

```diff
@@ -26,7 +26,8 @@
 """
 
 from time import sleep
-from adafruit_ht16k33.ht16k33 import HT16K33
+#from adafruit_ht16k33.ht16k33 import HT16K33
+from adafruit_ht16k33.matrix import HT16K33
```

Comment:
Why was this changed from the base class to the 8x8 matrix class?

Reply:
It makes no sense to me now either. Reversed